readme.md

```diff
@@ -19,17 +19,15 @@ Unlike Postgres, Firebase is a NoSQL database. This means that information is no
 #### Why Use Firebase?
 
 It's fast.
-* Firebase uses Websockets to maintain a constant, open connection between the client and the database. Rather than use a traditional call-and-return system of requests and responses like HTTP, the browser and server are constantly in communication with each other and are immediately aware of changes on either end. Using Firebase, we can establish three-way data binding in an Angular application between the controller, the browser and the database.
-
-> If you're interested in learning more about Websockets, [start here](http://www.html5rocks.com/en/tutorials/websockets/basics/).
+* Firebase uses Websockets to maintain a constant, open connection between the client and the database. Rather than use a traditional call-and-return system of requests and responses like HTTP, the browser and server are constantly in communication with each other and are immediately aware of changes on either end. Using Firebase, we can establish **three-way data binding** in an Angular application between the controller, the browser and the database.
 
```

Comment:
![This one goes to 11](https://upload.wikimedia.org/wikipedia/en/0/06/Spinal_Tap_-_Up_to_Eleven.jpg)